### PR TITLE
Fixed signatures of gmp_div, odbc_exec and unregister_tick_function

### PR DESCRIPTION
**Pull request by danielmarschall**
**gmp_div**:  Return type is GMP , not resource

**odbc_exec**: Return type is resource|false , not resource
Note that the "flags" argument will be removed in PHP 8.0

**unregister_tick_function**: argument is now called "callback" and type "callable" instead of "string"
Example: $callable=array(&$object, 'my_method') or $callable=array('Object', 'my_method')

### CODE REVIEW

File: resources/functionMap.php [hunk 1]
```diff
@@ -8012,7 +8012,7 @@
 'odbc_do' => ['resource', 'connection_id'=>'resource', 'query'=>'string', 'flags='=>'int'],
 'odbc_error' => ['string', 'connection_id='=>'resource'],
 'odbc_errormsg' => ['string', 'connection_id='=>'resource'],
-'odbc_exec' => ['resource', 'connection_id'=>'resource', 'query'=>'string', 'flags='=>'int'],
+'odbc_exec' => ['resource|false', 'connection_id'=>'resource', 'query'=>'string', 'flags='=>'int'], // Note: 'flags' was removed in PHP 8.0
```
Comment (ondrejmirtes) [inline resources/functionMap.php:8015]: The note is useless here, you can submit a change to https://github.com/phpstan/phpstan-src/blob/master/resources/functionMap_php80delta.php instead

File: resources/functionMap.php [hunk 2]
```diff
@@ -12639,7 +12639,7 @@
 'unixtojd' => ['int|false', 'timestamp='=>'int'],
 'unlink' => ['bool', 'filename'=>'string', 'context='=>'resource'],
 'unpack' => ['array|false', 'format'=>'string', 'data'=>'string', 'offset='=>'int'],
-'unregister_tick_function' => ['void', 'function_name'=>'string'],
+'unregister_tick_function' => ['void', 'callback'=>'callable'],
```
Comment (ondrejmirtes) [inline resources/functionMap.php:12642]: The parameter does not need to be renamed

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

File: resources/functionMap.php (hunk 2 above)
Comment (danielmarschall) [reply resources/functionMap.php:12642]: Okay, sounds good. I just looked in the old manual . In PHP 7, it was called "$function_name" , and in PHP 8 it was renamed to "$callback" (so it should be renamed in functionMap_php80delta.php then)

File: resources/functionMap.php (hunk 1 above)
Comment (danielmarschall) [reply resources/functionMap.php:8015]: Okay. Will you do it, or do I need to send a new pull request?

File: resources/functionMap.php (hunk 2 above)
Comment (ondrejmirtes) [reply resources/functionMap.php:12642]: This doesn't need to be done, the PHP 8 signatures are mostly taken from https://github.com/phpstan/php-8-stubs/blob/main/stubs/ext/standard/unregister_tick_function.php already.

File: resources/functionMap.php (hunk 1 above)
Comment (ondrejmirtes) [reply resources/functionMap.php:8015]: Modify this PR so it can be merged, thanks.

Comment (danielmarschall) [reply resources/functionMap.php]: Ok, I changed the PR

**Review by ondrejmirtes — APPROVED** [review]

**Comment by ondrejmirtes** [comment]
Thank you!

**Pull request merged**